src/d8/drupal.make.yml

```diff
@@ -0,0 +1,21 @@
+core: 8.x
+api: 2
+
+# Common attributes.
+defaults:
+  projects:
+    subdir: "contrib"
+
+projects:
+  drupal:
+    version: ~
+  features:
+    version: ~
+  field_group:
+    version: ~
+  panels:
+    version: ~
+  pathauto:
+    version: ~
+  token:
+    version: ~
```

Comment:
some of these definitely are not ready today, culling recommended. will see if i can help